.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
is this not the default?

Reply:
I believe that file is generated automatically by VS Code and should have not been committed & merged.